api/schemas/resource_type.json

```diff
@@ -1,43 +0,0 @@
-{
```

Comment:
Probably okay since none of the other Puppet API schemas refer back to this file directly, it seems?